store/storetest/group_supplier.go

```diff
@@ -1739,12 +1739,12 @@ func testGetGroupsByChannel(t *testing.T, ss store.Store) {
 			}
 			tc.Opts.PageOpts.Page = tc.Page
 			tc.Opts.PageOpts.PerPage = tc.PerPage
-			res := <-ss.Group().GetGroupsByChannel(tc.ChannelId, tc.Opts)
-			require.Nil(t, res.Err)
-			require.ElementsMatch(t, tc.Result, res.Data.([]*model.Group))
+			groups, err := ss.Group().GetGroupsByChannel(tc.ChannelId, tc.Opts)
+			require.Nil(t, err)
+			require.ElementsMatch(t, tc.Result, groups)
 			if tc.TotalCount != nil {
-				res = <-ss.Group().CountGroupsByChannel(tc.ChannelId, tc.Opts)
-				count := res.Data.(int64)
+				var count int64
```

Comment:
nit: the signature of `CountGroupsByChannel` is already specifying that the return type is `int64`, is it neccessary to create the variable before the function call?

Reply:
I did that to avoid the shadowing of the 'err' variable there. I'm just reusing the `err` variable.

Reply:
With the `:=` notation can be a bit ambiguous, so, I prefer this way.